packages/cli/src/virtual-routes/components/Html.tsx

```diff
@@ -0,0 +1,16 @@
+import {Links, Meta} from '@remix-run/react';
```

Comment:
There have been changes in the virtual routes, this file is not needed anymore. There's a [`virtual-root.tsx`](https://github.com/Shopify/h2/tree/2023-01/packages/cli/src/virtual-routes) now that adds the HTML where you can also add error boundaries etc. I think you should get it if you merge the branch `2023-01`.

Reply:
Whoops, forgot to do a last push!